app/src/main/java/edu/grinnell/appdev/grinnelldirectory/APICaller.java

```diff
@@ -0,0 +1,138 @@
+package edu.grinnell.appdev.grinnelldirectory;
+
+import android.util.Log;
+
+import java.io.IOException;
+import java.util.List;
+
+import edu.grinnell.appdev.grinnelldirectory.Interfaces.APICallerInterface;
+import edu.grinnell.appdev.grinnelldirectory.Interfaces.DatabaseAPI;
+import edu.grinnell.appdev.grinnelldirectory.Model.Person;
+import edu.grinnell.appdev.grinnelldirectory.Model.User;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+
+
+public class APICaller {
+
+    private static final String BASE_URL = "https://itwebappstest.grinnell.edu/DotNet/WebServices/api/";
+
+    private Retrofit retrofit;
+    private DatabaseAPI dbAPI;
+    private Call<List<Person>> personQuery;
+    private APICallerInterface apiCallerInterface;
+    private User user;
+
+    public APICaller(User user1, APICallerInterface apiInterface) {
+        retrofit = new Retrofit.Builder().
+                baseUrl(BASE_URL).
+                addConverterFactory(GsonConverterFactory.create()).
+                build();
+        dbAPI = retrofit.create(DatabaseAPI.class);
+        apiCallerInterface = apiInterface;
+        user = user1;
+    }
+
+    /* - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -  */
+    public void simpleSearch(User user, List<String> fields) {
+        personQuery = dbAPI.simpleSearch(user, fields.get(0), fields.get(1), fields.get(2), fields.get(3));
```

Comment:
What is 0, 1 ...3 here? It is not clear what these numbers are. It'll be better to have them as constant member variables with descriptive names and use them here. eg. you can do this:
`private static int FIRST_NAME_FIELD = 0` to represent 0 (I could only know this by having to read through the other classes. Let's make the code easily readable and maintainable)